resources/views/home.blade.php

```diff
@@ -1,5 +1,322 @@
+@php use Illuminate\Support\Facades\App; @endphp
 @extends("layout.public")
 
+@php
+    $caseStudiesFile = App::currentLocale()=='en' ? 'case_studies_en.json' : 'case_studies_pl.json';
+    $caseStudy = json_decode(file_get_contents(public_path($caseStudiesFile)), true)[0];
+    $testimonials = json_decode(file_get_contents(public_path('testimonials.json')), true);
+    $articles = json_decode(file_get_contents(public_path('articles.json')), true);
+
+@endphp
 @section("content")
-    🏠
+    <section class="relative overflow-hidden">
+        <div class="relative md:block -z-10">
+            <div class="absolute -left-[6%] md:left-[1%] top-28 md:top-40 -z-10">
```

Comment:
I think you don't need this z-index when you add the same above.

```suggestion
            <div class="absolute -left-[6%] md:left-[1%] top-28 md:top-40">
```